app/src/main/java/com/sunasterisk/weather/utils/WeatherUtils.kt

```diff
@@ -0,0 +1,55 @@
+package com.sunasterisk.weather.utils
+
+import kotlin.math.roundToInt
+import com.sunasterisk.weather.data.model.element.Direction
+import com.sunasterisk.weather.data.model.element.SpeedUnit
+import com.sunasterisk.weather.data.model.element.TemperatureUnit
+
+object WeatherUtils {
+
+    fun formatTemperature(vararg ints: Int?): String? {
+        if (ints.isNotEmpty()) {
+            return when (ints.size) {
+                1 -> "${ints[0]}°"
+                2 -> "${ints[0]}°/${ints[1]}°"
+                else -> null
+            }
+        }
+        return null
+    }
+
+    fun temperatureUnit(data: Double?, temperatureUnit: TemperatureUnit): Int? {
+        return when(temperatureUnit.unit) {
+            TemperatureUnit.CELSIUS.unit -> { data?.let { ((it - 32) / 1.8).roundToInt() } }
+            else -> data?.roundToInt()
+        }
+    }
+
+    fun formatWindDirection(data: Int?): String? {
+        return when(data) {
+            in 23..67 -> " ${Direction.NORTHEAST.direction}"
```

Comment:
```suggestion
            in 23..67 -> Direction.NORTHEAST.direction
```
Chú thích các giá trị luôn